Unix/tests/cli/test_cli.cpp

```diff
@@ -537,7 +537,7 @@ static int StopServerSudo()
 
 cleanup:
     // To allow pid file to be deleted
-    Sleep_Milliseconds(200);
+    Sleep_Milliseconds(500);
```

Comment:
Does this work on all systems (even slow ones, like HP)? Perhaps better to have a tighter loop with a long wait time, and drop out if/when the PID file disappears?